src/o365/spo/commands/site/site-list.ts

```diff
@@ -109,6 +116,10 @@ class SpoSiteListCommand extends SpoCommand {
       {
         option: '-f, --filter [filter]',
         description: 'filter to apply when retrieving sites'
+      },
+      {
+        option: '--deleted [deleted]',
```

Comment:
As we are not passing anything into this option we can just use `--deleted` and remove `[deleted]`